test/behaviour/concept/thing/attribute/BUILD

```diff
@@ -50,11 +50,11 @@ grakn_behaviour_java_test(
         "@graknlabs_behaviour//concept/thing:attribute.feature",
     ],
     grakn_core_artifact_mac = "@graknlabs_grakn_core_artifact_mac//file",
-    grakn_core_artifact_windows = "@graknlabs_grakn_core_artifact_windows//file",
     grakn_core_artifact_linux = "@graknlabs_grakn_core_artifact_linux//file",
+    grakn_core_artifact_windows = "@graknlabs_grakn_core_artifact_windows//file",
```

Comment:
The reason for all these odd-looking changes is a merge conflict, another PR made the same change as mine. Functionally there is no change.